prboom2/src/SDL/i_system.c

```diff
@@ -333,6 +333,15 @@ dboolean HasTrailingSlash(const char* dn)
           );
 }
 
+static const char *GetExecutableDir(void)
```

Comment:
What's wrong with `I_DoomExeDir()`?

Reply:
Unlike what the name seems to imply, the `I_DoomExeDir()` function returns the path to `~/.dsda-doom`, not the actual directory the executable is located in.